styles.css

```diff
@@ -112,6 +112,308 @@ section {
   gap: 20px;
 }
 
```

Comment:
- [ ] As requested by the previous reviewer about the headline section header please stick to the Figma design as 
 much possible 👏🏼 the header needs some room to breathe by increasing the `width` property that is explained in the below example 👍🏼 

| Your Design | Figma Design |
|---|---|
| ![image](https://user-images.githubusercontent.com/68030297/230506640-9cf40564-a290-4aa8-925c-a87917a40c33.png) | ![image](https://user-images.githubusercontent.com/68030297/230506488-3d2f36d6-2908-4842-8e84-069f9ccc5ee2.png) |

### Suggestions 🥇 
- From CSS checkout the selector `.headline .container` and try the following
```javascript
.headline .container {
    ...
    width: 90%
    ...
}